packages/core/src/App/Containers/Redirect/redirect.jsx

```diff
@@ -139,6 +139,19 @@ const Redirect = ({
             }
             break;
         }
+        case 'add_account_multiplier': {
+            WS.wait('get_account_status').then(() => {
+                if (!currency) return openRealAccountSignup('set_currency');
+                if (hasAnyRealAccount()) return openRealAccountSignup('manage');
+                return openRealAccountSignup('maltainvest');
```

Comment:
@henry-deriv i think this needs specific checks since we can have 2 different regulations in the same account. Did you try to test in every edge case ?

Reply:
Can you elaborate?

Reply:
Resolved. Talk to thisyahlen. No need to add checks.